hpOneView/oneview_client.py

```diff
@@ -1130,6 +1132,18 @@ def appliance_time_and_locale_configuration(self):
         return self.__appliance_time_and_locale_configuration
 
     @property
+    def versions(self):
+        """
+        Gets the Version API client.
+
+        Returns:
+            Version:
```

Comment:
dict: The minimum and maximum supported API versions.

Reply:
But this does not returns anything besides the client, you would still need to call get_version.

Reply:
RIGHT! I somehow placed this comment at the wrong place. This should refer to [line 54 of  versions.py](https://github.com/HewlettPackard/python-hpOneView/pull/334/files#diff-5d96eafc6a1679aae6b25203b94c9134R54)